simulator/examples/input-handling.rs

```diff
@@ -0,0 +1,38 @@
+//! # Example: Input Handling
+//!
+//! This example allows you to move a red circle to the location of a click on the simulator
+//! screen. Although input handling is not a part of the embedded-graphics API, the simulator can
+//! be used to emulate input controls in order to represent more complex UI systems such as touch
+//! screens.
+extern crate embedded_graphics;
+extern crate embedded_graphics_simulator;
+
+use embedded_graphics::pixelcolor::Rgb888;
+use embedded_graphics::prelude::*;
+use embedded_graphics::primitives::Circle;
+use embedded_graphics_simulator::DisplayBuilder;
+
+fn main() {
+    let mut display = DisplayBuilder::new().size(800, 480).build_rgb();
```

Comment:
Could you add some window title like: `Click to move the circle`. This would help if someone starts the example without reading it first.